image-loader/conf/routes

```diff
@@ -1,5 +1,7 @@
 GET     /                           controllers.ImageLoaderController.index
-POST    /images                     controllers.ImageLoaderController.loadImage(uploadedBy: Option[String], identifiers: Option[String], uploadTime: Option[String], filename: Option[String])
+POST    /images                     controllers.ImageLoaderController.loadImage(uploadedBy: String, identifiers: Option[String], uploadTime: Option[String], filename: Option[String])
+# Not exposed in UI - force complete reload of file already correctly stored in S3.
+GET     /imageReload                controllers.ImageLoaderController.reloadImage(uploadedBy: Option[String], identifiers: String)
```

Comment:
Should this be a POST instead?

Reply:
As I said to Simon, there is a very good reason for this....  I'll discuss before merge.